modules/gitops-release-notes-1-1.adoc

```diff
@@ -0,0 +1,129 @@
+* Previously, the {gitops-title} backend failed to find the valid GitLab repository. This was because the {gitops-title} backend checked for `main` as the branch reference, instead of `master` in the GitLab repository. This issue is fixed now. link:https://issues.redhat.com/browse/GITOPS-768[GITOPS-768]
+* The *Environments* page in the {product-title} web console from the *Developer* perspective now shows the list of applications and the number of environments along with an Argo CD link that directs you to the Argo CD *Applications* page that lists all the applications. The Argo CD *Applications* page has *LABELS* (for example, `app.kubernetes.io/name=appName`) that helps you filter only the applications of your choice to be shown in that view. link:https://issues.redhat.com/browse/GITOPS-544[GITOPS-544]
+
+
+[id="known-issues-1-1_{context}"]
+== Known issues
+These are the known issues in {gitops-title} 1.1:
+
+* {gitops-title} does not support Helm v2 and ksonnet.
+* Red Hat SSO (RH SSO) Operator is not supported in disconnected clusters. As a result, the {gitops-title} Operator and RH SSO integration is not supported in disconnected clusters.
+* When you delete an Argo CD application from the {product-title} web console, the Argo CD application gets deleted in the user interface, but the deployments are still present in the cluster. As a workaround, delete the Argo CD application from the Argo CD console. link:https://issues.redhat.com/browse/GITOPS-830[GITOPS-830]
+
+
+[id="breaking-change-1-1_{context}"]
+== Breaking Change
+=== Upgrading from {gitops-title} v1.0.1
+
+When you upgrade from {gitops-title} `v1.0.1` to `v1.1`, the {gitops-title} Operator renames the default Argo CD instance created in the `openshift-gitops` namespace from `argocd-cluster` to `openshift-gitops`.
+
+This is a breaking change and needs the following steps to be performed manually, before the upgrade:
+
+. Go to the {product-title} web console and copy the content of the `argocd-cm.yml` config map file in the `openshift-gitops` namespace to a local file. The content may look like the following example:
++
+.Example argocd config map YAML
+[source,yaml]
+----
+    kind: ConfigMap
+    apiVersion: v1
+    metadata:
+     selfLink: /api/v1/namespaces/openshift-gitops/configmaps/argocd-cm
+     resourceVersion: '112532'
+     name: argocd-cm
+     uid: f5226fbc-883d-47db-8b53-b5e363f007af
+     creationTimestamp: '2021-04-16T19:24:08Z'
+     managedFields:
+    ...
+     namespace: openshift-gitops
+     labels:
+       app.kubernetes.io/managed-by: argocd-cluster
+       app.kubernetes.io/name: argocd-cm
+       app.kubernetes.io/part-of: argocd
+       data:                                            "" <1>
```

Comment:
`data:` should line up with `metadata:`.  Everything else looks good.

Reply:
Done

Reply:
Done